docs/how-to-guides/how-to-create-a-new-service.md

```diff
@@ -8,12 +8,9 @@ This guide will help you in the steps to create a new service for the
 If you have not already, we highly recommend you to follow the two official
```

Comment:
Maybe it's "three official tutorials" ?

Reply:
I'd suggest to change it to "official tutorials" without any count (or "other tutorials"), to ease maintenance in case more tutorials are added later on.